packages/taquito-beacon-wallet/README.md

```diff
@@ -26,23 +26,27 @@ import { BeaconWallet } from '@taquito/beacon-wallet';
 const options = {
   name: 'MyAwesomeDapp',
   iconUrl: 'https://taquito.io/img/favicon.svg',
-  preferredNetwork: 'chosen_network',
+  network: { type: 'ghostnet' },
   eventHandlers: {
```

Comment:
wasn't sure how relevant is `eventHandlers` now there is `subscribeToEvent` in place